packages/block-editor/src/components/block-card/stories/index.story.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import { box, button, cog, paragraph } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import BlockCard from '../';
+
+export default {
+	title: 'BlockEditor/BlockCard',
+	component: BlockCard,
+	parameters: {
+		docs: {
+			description: {
+				component:
+					'The `BlockCard` component allows to display a card which contains the title of a block, its icon and its description.',
+			},
+			canvas: { sourceState: 'shown' },
+		},
+	},
+	argTypes: {
+		title: {
+			control: 'text',
+			description: 'The title of the block',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		description: {
+			control: 'text',
+			description: 'A description of the block functionality',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		icon: {
+			control: 'select',
+			options: [ 'paragraph', 'cog', 'box', 'button' ],
+			mapping: {
+				paragraph,
+				cog,
+				box,
+				button,
+			},
+			description: 'The icon to display for the block',
+		},
+		name: {
+			control: 'text',
+			description: 'Optional custom name for the block',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		className: {
+			control: 'text',
+			description: 'Additional CSS class names',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+	},
+};
+
+/**
+ * Default story shows the basic BlockCard with title, icon, name and description.
+ */
+export const Default = {
```

Comment:
```suggestion
export const Default = {
```

Since there is only one story, we may be able to delete this comment.

Reply:
Removed the redundant comment.